middlewares/authorization.js

```diff
@@ -0,0 +1,18 @@
+import jwt from 'jsonwebtoken';
+
+const authenticateUserToken = (req, res, next) => {
+    const authHeader = req.headers['authorization'];
```

Comment:
How do you ensure that this will work? It might be a good idea to introduce a try/catch block here.